spec/requests/search_spec.rb

```diff
@@ -23,6 +23,15 @@
     end
   end
 
+  describe "ui" do
```

Comment:
I think it's great that we have this test. Might be easier to find if we put it in a separate file that corresponds with the path. similar to `spec/requests/api/v0/post_manifest_spec.rb` we could have the other request specs at `spec/requests/api/v1/manifests_search_spec.rb` and then have this test in a file at `spec/requests/manifest_search_spec.rb`..or something like that. 

What do you think?